_data/changelogs/component-range-slider.yml

```diff
@@ -2,6 +2,17 @@ title: Range slider
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Added optional `data-text-unit` and `data-text-preposition` attributes.
+  summaryAdditional: These optional attributes provide additional context to screen reader users, and allow for proper preposition translations for range sliders on pages in languages other than English.
```

Comment:
```suggestion
  summaryAdditional: The `data-text-unit` attribute provides additional context to screen reader users, and `data-text-preposition` allows for proper preposition translations on non-English pages.
```

Reply:
Resolved in [16a0ea7](https://github.com/uswds/uswds-site/pull/2310/commits/16a0ea7739e49bfa25480dc73bca6931890624d9)